pulp_ansible/app/tasks/collections.py

```diff
@@ -131,10 +131,34 @@ def import_collection(artifact_pk):
             collection, created = Collection.objects.get_or_create(
                 namespace=collection_info["namespace"], name=collection_info["name"]
             )
-            collection_version = CollectionVersion(
-                collection=collection, version=collection_info["version"]
-            )
+            if created:
+                CreatedResource.objects.create(content_object=collection)
+
+            dependencies = collection_info.pop('dependencies')
+            tags = collection_info.pop('tags')
+
+            # Mazer returns many None values. We need to let the default='' in models.py prevail
```

Comment:
The importer currently also returns these as `null`/`None` so this will continue to be needed